.vale/styles/Google/Units.yml

```diff
@@ -2,7 +2,7 @@ extends: existence
 message: "Put a nonbreaking space between the number and the unit in '%s'."
 link: "https://developers.google.com/style/units-of-measure"
 nonword: true
-level: error
+level: warning
 tokens:
```

Comment:
I'm down with dropping this rule for now if you'd like. Feels unimportant ultimately